v3/tutorials/11-kitties-workshop/b-kitties-frontend/index.mdx

```diff
@@ -108,7 +108,7 @@ We'll be building out a total of 3 components:
 
 1. `Kitties.js`: this will render the Kitty pane, and contains the logics of fetching all kitties information from the connecting Substrate node.
 
-1. `KittyCards.js`: this will render a React card component containing a Kitty's relevantinformation, avatar and buttons to interact with it.
+1. `KittyCards.js`: this will render a React card component containing a Kitty's relevantin formation, avatar and buttons to interact with it.
```

Comment:
```suggestion
1. `KittyCards.js`: this will render a React card component containing a Kitty's relevant information, avatar and buttons to interact with it.
```